src/lighting/lighting-controller.js

```diff
@@ -400,6 +407,29 @@ class LightingController {
   }
 
   /**
+   * Uses the Hue API to change the color of all lights within range.
+   */
+  colorLights(xColor) {
+    var _self = this;
+
+    switch(xColor){
```

Comment:
It may be better for maintainability to set the RGB value to a local variable within this _switch_ statement, and then just have the complex API command (_Hue.lightState.create().on().hue(rgbValue).sat(255)_) appear once after the _switch_ statement.